include/envoy/api/os_sys_calls_hot_restart.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifndef WIN32
+#include <sys/mman.h> // for mode_t
+
```

Comment:
nit: del line

Reply:
hmmm clang-format adds it back... same as https://github.com/envoyproxy/envoy/blob/master/include/envoy/api/os_sys_calls.h#L9

Reply:
No problem.